Lib/distutils/command/install.py

```diff
@@ -280,7 +280,14 @@ def finalize_options(self):
         # $platbase in the other installation directories and not worry
         # about needing recursive variable expansion (shudder).
 
-        py_version = sys.version.split()[0]
+        releaselevel_serial = ''
+        if sys.version_info[3] == 'alpha' or sys.version_info[3] == 'beta':
```

Comment:
I prefer to use the `in` operator
```suggestion
        if sys.version_info[3] in ('alpha', 'beta'):
```